### PR TITLE
Updated Türkçe(Turkish) translation

### PR DESCRIPTION
**Pull request by mertcangokgoz**
Türkçe(Turkish) language support has been updated.

### CODE REVIEW

File: ghost/i18n/locales/tr/portal.json [hunk 1]
```diff
@@ -25,12 +25,12 @@
     "Cancel subscription": "Aboneliği iptal et",
     "Cancellation reason": "İptal sebebi",
     "Change": "Değiştir",
-    "Change plan": "",
+    "Change plan": "Plan Değiştir",
```
Comment (cathysarisky) [inline ghost/i18n/locales/tr/portal.json:28]: Is this (lack of) spacing and the capitalization correct?

File: ghost/i18n/locales/tr/portal.json [hunk 2]
```diff
@@ -25,12 +25,12 @@
...
     "Check spam & promotions folders": "Spam ve promosyonlar klasörlerini kontrol edin",
     "Check with your mail provider": "Posta sağlayıcınızla kontrol edin",
     "Choose": "Seç",
     "Choose a different plan": "Farklı bir plan seç",
-    "Choose a plan": "",
+    "Choose a plan": "Bir Plan Seçin",
```
Comment (cathysarisky) [inline ghost/i18n/locales/tr/portal.json:33]: Is this (lack of) spacing and the capitalization correct?

**Review by cathysarisky — CHANGES_REQUESTED** [review]
Hey @mertcangokgoz , thank you for these!  It looks like there are a few strings that are still in English.  Are you able to add those as well?

File: ghost/i18n/locales/tr/portal.json (hunk 2 above)
Comment (cathysarisky) [reply ghost/i18n/locales/tr/portal.json:33]: Sorry - spacing looks perfectly fine here, not sure what was up on my other screen.  The question about capitalization remains.

Comment (mertcangokgoz) [reply ghost/i18n/locales/tr/portal.json]: Thank you for your notification, I have made the correction.

File: ghost/i18n/locales/tr/portal.json (hunk 1 above)
Comment (cathysarisky) [inline ghost/i18n/locales/tr/portal.json:28]: Sorry, one more with the same capitalization quirk.

Comment (mertcangokgoz) [reply ghost/i18n/locales/tr/portal.json]: fixed

**Review by cathysarisky — APPROVED** [review]
Thanks!

**Comment by ronaldlangeveld** [comment]
Thank you for submitting translations to Ghost! 🙏🏻 We really appreciate every string you’ve added. There might be a few new strings that need translation in Turkish to ensure your site. If you spot any untranslated or missing strings, let us know, and we’ll fix them ASAP! 🎉

**Pull request merged**